### ISSUE TITLE
"GIST: Open Block" opens more than the last 100 gists [enhancement]

### ISSUE THREAD

**Issue by vctsvz** [kenhowardpdx/vscode-gist#45]
FEATURE:

**This is a**

- [ ] bug report
- [x] feature request

#### Description
Add the ability to load with `"GIST: Open Block"` more than the last **100** gists.
#

**Comment by kenhowardpdx** [comment]
Yes. I'm not certain how because I believe the GitHub REST API returns a max of 100 then provides pagination paths.

**Comment by AndryWJ** [comment]
can receive all pages? or flipping through the pages until a match is found, slowly but at least it will work
